src/_pages/prototyping_boards/electronic-use.md

```diff
@@ -68,7 +68,39 @@ We created an open-source **Python library** managing this JSON API called [*Pyl
 
 Get <a href="https://github.com/Luos-io/Pyluos" target="_blank">pyluos on github</a>.
 
-## Update board's firmware
+## Update Luos, Robus and board's firmware
+
+To Update Luos, Robus and module's code you must go on the librairies window of platformio in Visual Studio Code:
+        <img src="{{img_path}}/Update_Luos_Robus_Modules.png" height="" />
+
+
+To update the board's firmware you need to follow 3 steps:
+
+    1. Open the module folder that you want to update in Visual Studio Code
+            To do this, you must go to "file" -> "open the folder" and go to the folder of the module you want
+
+    2. Compile it to detect if there is an error
+            To compile your code, you must click on this button at the bottom of your window:
+<img src="{{img_path}}/compile.png" height="" />
+
+            If your terminal goes like this, it means that your download was successful:
+<img src="{{img_path}}/compile_success.png" height="" />
+
+            If the compilation goes wrong and you don't know why, because your code has been perfectly compiled before and you change
+            nothing, you can do a cleaning by clicking on the ant button and choosing Clean in the list of project tasks:
+<img src="{{img_path}}/clean.png" height="" />
+
+
+    3. Upload the new code into the board
+            To upload the new code into the board you must to click on this button:
+<img src="{{img_path}}/upload.png" height="" />
+
+            If your terminal goes like this, it means that your download was successful:
+<img src="{{img_path}}/upload.png" height="" />
+
+            If the compilation goes wrong and it may be because your driver is not updated specially on windows, if this is the case you
```

Comment:
```suggestion
            If the compilation goes wrong and it may be because your driver is not updated specially on Windows, if this is the case you
```